src/lib/verifier/DAVerifier.sol

```diff
@@ -236,9 +236,12 @@ library DAVerifier {
     /// @param _end The ending of the range (exclusive).
     /// @return _ the sliced data.
     function slice(bytes[] memory _data, uint256 _begin, uint256 _end) internal pure returns (bytes[] memory) {
+        if (_begin > _end) {
+            revert("Invalid range: _begin is greater than _end");
+        }
```

Comment:
Hmmm did we want library code to revert vs returning some sort of error?

Reply:
[e2daa12](https://github.com/celestiaorg/blobstream-contracts/pull/254/commits/e2daa12b23501802cba0ad6c5147df479203d1af)